code/modules/mob/living/simple_animal/hostile/bees.dm

```diff
@@ -295,3 +295,12 @@
 			forceMove(beehome.drop_location())
 	else
 		..()
+		
+/mob/living/simple_animal/hostile/poison/bees/short
+	desc = "These bees seem unstable and won't survive for long."
+
+/mob/living/simple_animal/hostile/poison/bees/short/suicide()
+	health = 0
+
+/mob/living/simple_animal/hostile/poison/bees/short/Initialize()
+	addtimer(CALLBACK(src, .proc/suicide), 500)
```

Comment:
You can replace the `500` here with `50 SECONDS`

That seems a little long to me but we can tone it down if it becomes an issue.